recipes-connectivity/connman/connman/0001-main-add-ve-and-vb-to-the-default-interface-blacklis.patch

```diff
@@ -0,0 +1,32 @@
+From 85c6b7880473e72196203160c4057341c6cad20a Mon Sep 17 00:00:00 2001
+From: Krisztian Litkey <krisztian.litkey@intel.com>
+Date: Thu, 31 Mar 2016 15:14:51 +0300
+Subject: [PATCH 1/1] main: add ve- and vb- to the default interface blacklist.
+
+Add ve- and vb- to the list of default blacklisted interface
+prefixes. Such interfaces are usually used by systemd-nspawned
+containers and are typically managed by systemd-networkd.
+
+Upstream-Status: Pending
+
+Signed-off-by: Krisztian Litkey <krisztian.litkey@intel.com>
+---
+ src/main.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/src/main.c b/src/main.c
+index e46fa7b..f44a2ed 100644
+--- a/src/main.c
++++ b/src/main.c
+@@ -57,6 +57,8 @@ static char *default_blacklist[] = {
```

Comment:
@pfl Thanks!

@klihub we should add the corresponding patch to meta-ostro-os-fixes as "Upstream-Status: Backported" (but let's check with mythi) and drop the main.conf from meta-appfw (as done in a latter commit in this PR).

@mythi do you agree that meta-ostro-os-fixes is a better place?

@juimonen once this merged, we'll no longer have /etc/connman/main.conf and we would be able to drop our non-upstreamable ConnMan patches from the stateless branch, after archiving them somewhere in case that they'll be needed again later for some reasons.